### DISCUSSION TITLE
Add aria-current variant

### DISCUSSION THREAD

**Discussion by MichaelAllenWarner** [tailwindlabs/tailwindcss#9563] [Ideas]
The new built-in [`aria-` variants](https://github.com/tailwindlabs/tailwindcss/pull/9557) are great!

I wonder if it might make sense to add one for [`aria-current`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current). Although it isn't a boolean attribute, it almost always functions like one in practice. What I mean is: probably ~99% of the time, a given element should either have one specific non-`"false"` value (like `"page"`) or have a `"false"`/missing value; rarely (if ever) might an element have more than one possible non-`"false"` value.

So for styling purposes, a selector like `[aria-current]:not([aria-current="false"])` might suffice. (I've been using that as a custom Tailwind variant for some time now.)

**Comment by Flowko** [comment]
for anyone finding this, you can use https://www.npmjs.com/package/tailwindcss-aria-attributes
